docker-compose.yml

```diff
@@ -71,6 +71,10 @@ services:
         MAPBOX_INIT_LATITUDE: ${MAPBOX_INIT_LATITUDE}
         MAPBOX_INIT_LONGITUDE: ${MAPBOX_INIT_LONGITUDE}
         MAPBOX_INIT_ZOOM: ${MAPBOX_INIT_ZOOM}
+        CVIZ_MAPBOX_TOKEN: ${CVIZ_MAPBOX_TOKEN}

```

Comment:
Does this need to exist separate from the MAPBOX_TOKEN above?

Reply:
Removed this requirement